frontend/app/login/page.jsx

```diff
@@ -11,21 +11,70 @@ const Login = () => {
 
     const signIn = async () => {
         try {
-            const { user: { uid } } = await signInWithPopup(auth, provider);
+            const {
+                user: { uid },
+            } = await signInWithPopup(auth, provider);
             Cookies.set('uid', uid);
-            router.push('/dashboard'); 
-        } catch(err) {
+            router.push('/teams');
+        } catch (err) {
             console.log(err);
         }
-    }
-    
+    };
+
     return (
         <div className={loginStyles.container}>
-            <button onClick={() => signIn()}>
-                 { `G -> Sign in with Google` }
-            </button>
+            <div className={loginStyles.blurbBackground}>
+                <div className={loginStyles.blurbContainer}>
+                    <div className={loginStyles.headerContainer}>
+                        <h1 className={loginStyles.headerAssistant}>Assistant</h1>
+                        <div className={loginStyles.underline}></div>
+                        <h1>Captain</h1>
+                    </div>
+                    <p>
+                        The app that let's you create teams and manage rosters. Send SMS to confirm
+                        upcoming game rosters. View team schedules and build rosters according to
+                        your needs.
+                    </p>
+                </div>
+            </div>
+            <div className={loginStyles.loginContainer}>
+                <div className={loginStyles.login}>
+                    <h4>{`Login :)`}</h4>
+                    <button onClick={() => signIn()}>
+                        <svg
```

Comment:
I would definitely create a separate file for this SVG. It can make code look more polluted very quickly.